src/test/librbd/object_map/test_mock_InvalidateRequest.cc

```diff
@@ -44,7 +44,7 @@ TEST_F(TestMockObjectMapInvalidateRequest, UpdatesInMemoryFlag) {
     std::unique_lock image_locker{ictx->image_lock};
     request->send();
   }
-  ASSERT_EQ(0, cond_ctx.wait());
+  ASSERT_EQ(-EROFS, cond_ctx.wait());
```

Comment:
Hmm -- where is `-EROFS` coming from? I would want this test to expect no error. Same comment applies below.

Reply:
@dillaman  It comes from [L56](https://github.com/ceph/ceph/blob/master/src/librbd/object_map/InvalidateRequest.cc#L56) (since error have propagated back). 
I passed a param of [true](https://github.com/ceph/ceph/pull/34059/commits/f1386ebc0fc349405f7579584054df1255788dee#diff-def0606534f993f6de4ad41c2ece775cR36) so that it doesn't return an error, but i'm not sure if you like it, please take a look.  -- All updated. Thank you.